src/ocean/util/log/StatsReader.d

```diff
@@ -215,13 +216,56 @@ unittest
 
 public class StatsLogReader
 {
+    /**************************************************************************
+
+        Struct used to iterate through valid stat lines
```

Comment:
I suggest we extend documentation to say that it's silently ignoring empy stat lines, and that no other validity check (such as if the stat line is well formed) is performed, as "valid stat lines" is not really true - you can have non-empty but broken stats line.

Reply:
done